susemanager-utils/testing/docker/scripts/push-to-obs.sh

```diff
@@ -52,7 +52,7 @@ cd ${REL_ENG_FOLDER}
 
 # If we have more than one destinations, keep SRPMS so we don't
 # need to rebuild for each submission
-if [ "$(echo ${DESTIONATIONS}|cut -d',' -f2)" != "" ]; then
+if [ "$(echo ${DESTINATIONS}|cut -d',' -f2)" != "" ]; then
```

Comment:
This is just a minor typo that removed the the packages in all cases, but we want to keep them if there are more than one destinations.